MAUI/Shimmer/getting-started.md

```diff
@@ -125,4 +125,39 @@ SfShimmer shimmer = new SfShimmer()
 
 ![Circle persona shimmer view in .NET MAUI.](images/overview/maui-circle-persona.gif)
 
+## Loading shimmer content
+
+Shimmer content is loaded when the [`IsActive`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Shimmer.SfShimmer.html#Syncfusion_Maui_Shimmer_SfShimmer_IsActive) property of [`SfShimmer`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Shimmer.SfShimmer.html), is disabled.
```

Comment:
Shimmer content is loaded when the [`IsActive`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Shimmer.SfShimmer.html#Syncfusion_Maui_Shimmer_SfShimmer_IsActive) property of the [`SfShimmer`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Shimmer.SfShimmer.html) is disabled.